project.clj

```diff
@@ -4,7 +4,20 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.9.0-alpha17" :scope "provided"]
+                 [org.clojure/clojurescript "1.9.908"]
                  [org.clojure/spec.alpha "0.1.123"]
                  [circleci/clj-yaml "0.5.5"]
-                 [cheshire "5.7.1"]]
-  :profiles {:dev {:dependencies [[org.clojure/test.check "0.9.0"]]}})
+                 [cheshire "5.7.1"]
+                 [cljs-node-io "0.5.0"]
```

Comment:
I think this is only needed in tests, so could you move it to dev profile dependencies?